core/src/main/java/software/amazon/awssdk/core/util/CompletableFutures.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright 2010-2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+package software.amazon.awssdk.core.util;
+
+import java.util.concurrent.CompletableFuture;
+
+/**
+ * Utility class for working with {@link CompletableFuture}.
+ */
+public final class CompletableFutures {
```

Comment:
I thought we have a checkstyle rule to validate against plural name.

Reply:
I believe that only applies to enums and enum-like classes.

cc: @millems

Reply:
Yes, the rule only applies to enums or classes with public static fields.

Reply:
Done